### ISSUE TITLE
payfordata tx not included generates a lot of log

### ISSUE THREAD

**Issue by mindstyle85** [celestiaorg/celestia-app#442]
## Summary of Bug

```
May 24 15:25:26 val01-cel-d celestia-appd[164734]: 3:25PM INF executed block height=139 module=state num_invalid_txs=0 num_valid_txs=0
May 24 15:25:26 val01-cel-d celestia-appd[164734]: 3:25PM INF commit synced commit=436F6D6D697449447B5B3133203530203637203139352037203233392031303320323530203136322031343620313434203820323437203135372033203130352039372033362039362038342039382039342033203839203234372036362032353120393820323532203232302035332034345D3A38427D
May 24 15:25:26 val01-cel-d celestia-appd[164734]: 3:25PM INF committed state app_hash=0D3243C307EF67FAA2929008F79D036961246054625E0359F742FB62FCDC352C height=139 module=state num_txs=0
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=2fd76fae32f587eceb266dce19053b20fce4e846 tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=7145da826bbf64f06aa4ad296b850fd697a211cc tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=1d6a3c3d9ffc828b926f95592e15b1b59b5d8175 tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=ec072065bd4c6126a5833c97c8eb2d4382db85be tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=fe2025284ad9517ee6e8b027024cf4ae17e320c9 tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=5a4c337189eed845f3ece17f88da0d94c7eb2f9c tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=5273f0deefa5f9c2d0a3bbf70840bb44c65d835c tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=4ff8ead2e28ed0255bb02054a11c0dbfa1e00e73 tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:29 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=72b34325513863152269e781d9866d1ec4d6a93a tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:30 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=cd1524191300d6354d6a322ab0bca1d7c8ddfd01 tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:30 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=edba881755c597c5641b8d8764d389bea7496edc tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=45d0154bea2e0bbffec343894072f5feab19d242 tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM INF Timed out dur=4994.420185 height=140 module=consensus round=0 step=1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=5273f0deefa5f9c2d0a3bbf70840bb44c65d835c tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=ec072065bd4c6126a5833c97c8eb2d4382db85be tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=7145da826bbf64f06aa4ad296b850fd697a211cc tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=1d6a3c3d9ffc828b926f95592e15b1b59b5d8175 tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=45d0154bea2e0bbffec343894072f5feab19d242 tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=fe2025284ad9517ee6e8b027024cf4ae17e320c9 tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=5a4c337189eed845f3ece17f88da0d94c7eb2f9c tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=cd1524191300d6354d6a322ab0bca1d7c8ddfd01 tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=72b34325513863152269e781d9866d1ec4d6a93a tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=2fd76fae32f587eceb266dce19053b20fce4e846 tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:31 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=4ff8ead2e28ed0255bb02054a11c0dbfa1e00e73 tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:33 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=fcff172744c51684aaefc6fd3433eae275a2f31b tx=66D53C2F9669E721ABB6CB6F6A768929838A7F568E8F6330E6AF9AE65389B5B9 version=v1
May 24 15:25:33 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=fcff172744c51684aaefc6fd3433eae275a2f31b tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:33 val01-cel-d celestia-appd[164734]: 3:25PM ERR checktx failed for tx err="tx already exists in cache" module=mempool peer=edba881755c597c5641b8d8764d389bea7496edc tx=327F34FA522BAFC4A93CA8D860BB40BF3A719410F8BB80E09CEEB8D54F301BF0 version=v1
May 24 15:25:34 val01-cel-d celestia-appd[164734]: 3:25PM INF Timed out dur=3000 height=140 module=consensus round=0 step=3
May 24 15:25:35 val01-cel-d celestia-appd[164734]: 3:25PM INF Timed out dur=1000 height=140 module=consensus round=0 step=7
May 24 15:25:35 val01-cel-d celestia-appd[164734]: 3:25PM INF received proposal module=consensus proposal={"Type":32,"block_id":{"hash":"5639F595849CA9569F5B218CF05E91844224CE333600F06B062C824077D8169F","parts":{"hash":"FEB8C6482114BCE69EEC1DC23AB4E566939021B9E4C0E3BDC0277D6625DBD235","total":1}},"height":140,"pol_round":-1,"round":1,"signature":"8jHnsrgjjXwZJJr6J7H+y4SWWIToXyapvZiTbBkObb3zX0ZZ2rIZvnpsBWWQRUtrvAs8niN01S8xgW+3r7nxCg==","timestamp":"2022-05-24T13:25:35.941058184Z"}
May 24 15:25:35 val01-cel-d celestia-appd[164734]: 3:25PM INF received complete proposal block hash=5639F595849CA9569F5B218CF05E91844224CE333600F06B062C824077D8169F height=140 module=consensus
```

## Version

v0.5.0

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by evan-forbes** [comment]
I think is still caused by a few pay for message txs being stuck in the mempool, and then they are being gossiped by all of our peers. That's why the tx hashes are the same, and the the peer ids are different. 

The txs aren't getting included in a block likely because they were created before the v0.5.1 binary was released.

If this is true, then we should be able to clear them out of the mempool a few different ways. 
- produce blocks with square sizes that were signed over (probably one of 16,32,64,128)
- have all of the validators change their configs to remove txs from the mempool after a period of time (like setting `ttl-num-blocks` or `ttl-duration`
- everyone restart their nodes at the same time, clearing network's collective mempool cache (this isn't actually an option but it would technically solve it I think)

**Comment by evan-forbes** [comment]
part of the reason that these logs are generated are because nodes are falling behind in the network. If there aren't receiving up to date blocks, then the txs will not get removed from the mempool and they will continue to gossip them to all of their peers.

**Comment by evan-forbes** [comment]
closing with v0.6.0 since we were able to pull in the fix from v0.35.7 and patch the rest ourselves
